cli/src/main.rs

```diff
@@ -246,3 +252,23 @@ fn check_parse_errors(parsed_crates: &HashMap<CrateName, ParsedData>) -> anyhow:
         Ok(())
     }
 }
+
+#[cfg(feature = "go")]
+fn is_package_present(options: &ArgMatches) -> bool {
```

Comment:
would it be slightly more readable to have a function that returns `anyhow::Result<String>` and call it on main line 219?

`    if let Some(go_package) = options.value_of(args::ARG_GO_PACKAGE) {`

here we could error out if the go package is not defined

Reply:
Edited the function so we check directly here.